src/api/treeTrackerApi.js

```diff
@@ -68,51 +71,75 @@ export default {
       handleError(error);
     }
   },
-  approveCaptureImage(id, morphology, age, captureApprovalTag, speciesId) {
```

Comment:
Let's keep `age` for now until we get round to refactoring the UI. We don't want to drop any information the user has entered.